src/dialogs/PaymentErrorDialog.vue

```diff
@@ -105,3 +104,14 @@ export default class PaymentErrorDialog extends Vue {
   }
 }
 </script>
+
+<style scoped>
+  .flex-shrink-0.bullet {
+    padding-left: 1.4px;
+    padding-right: 5px;
+  }
+
+  ::v-deep .v-icon {
+    font-size: 18px;
+  }
+</style>
```

Comment:
Comparing with the `<li>` tag bullets, the bullet is a bit too large and the alignment is off by a little, so I used the style to make sure our custom v-icon look exactly like the `<li>` bullets.

Reply:
1. Although you have matched the previous look exactly, these styles are now different from all other bulleted lists in this UI, which I discourage since we're trying for consistency.

2. Please have a conversation with UI/UX (Tracey or Yui) on whether these styles are OK. For one thing, I don't think we've ever used decimal or odd padding values before.

3. Please don't use the Vuetify layout classes for selecting elements. I consider these layout-only classes. One way to look at it is that you wouldn't want all "flex-shrink-0" classes to have this padding. But the classname you have assigned (bullet) is good.

4. Don't override the icon font size. That's the point of specifying a small, regular or large icon name. Or you can use small/large/etc in the [v-icon](https://vuetifyjs.com/en/api/v-icon/#props).

Reply:
![Screenshot 2022-04-25 103855](https://user-images.githubusercontent.com/51458187/165144032-ec72abc5-6748-404e-9a06-b4124eee5ed9.png)

![Screenshot 2022-04-25 103929](https://user-images.githubusercontent.com/51458187/165144055-40f02ea5-03d9-40f1-a05d-7f1e8b164202.png)

For displaying purposes, I put the error message together with the warning message. Since the warning message is using the `<li>` tags, I want to use the scoped style to make sure everything in our custom flex box `div` tag are the same as the `<li>` tag. Overriding the classes and tags are bad, and I will use `id` for these tags.

Reply:
I'm not sure what you're doing but I'll review your changes.

Don't overthink this. Do follow existing code styles/practises. Do check with UI/UX team on this.

Reply:
I unresolved this conversation because my 4 points above have not been addressed yet.

Reply:
Do you want to remove the scoped style?

Reply:
Here are my answers to the [questions](https://github.com/bcgov/business-create-ui/pull/384#discussion_r857845181):

1. I added the scoped style so that the bullets in both the error messages and the warning messages are the same size. Removing any of those styles will have different looks. If we want to be consistent with all other `v-icon` and `.flex-shrink-0` components, then we should remove the scoped style. By doing so, the resulted image is the second image.

2. I am still waiting for responses from the UI team. I will update this after they confirm with me.

3. I am using classes "error-bullet" and "error-circle" for the styles in my latest commit 6067d62.

4. Using small/large/etc in the [v-icon](https://vuetifyjs.com/en/api/v-icon/#props) doesn't look the same as the bullet for `<li>`. If we don't use style, then the current implementation `mdi-circle-medium` is the best size.

Reply:
1. Oh, I understand why you're trying to get both lists the same size: it's because the first list uses v-html and a MDI icon, and the second list still uses UL and LI elements. I think both lists should use v-html.

Reply:
2. If you've waited more than a day then bring this up in Standup.

Reply:
![image](https://user-images.githubusercontent.com/51458187/165157596-681a46c3-f1fd-4824-864b-c01fdda61506.png)

@severinbeauvais While Tracy is waiting for Scott to review the above image I sent to her, she spotted an interesting observation: "One question - I am wondering why we have contact information listed twice with 2 different sets of contact info? I know for sure we wouldn’t want that. And what is the “Please note the following warnings” information all about? It just seems like a lot of information that isn’t super helpful to the user.". That is, we are displaying the phone number, fax number and email twice in this error dialog.

Reply:
Comment on [1](https://github.com/bcgov/business-create-ui/pull/384#discussion_r857939481):

I agree with both should use `v-html`, but the `warnings` and 'errors' have different structures. 'warnings` is an array of dictionaries `[{message: '...'}, {message: '...'}]`, whereas 'errors' is an array of dictionaries with `[{message: '...', payment_error_type: '...' }]. It is the errors `message` key that contains stringified html, so v-html only applies to the errors, not the warnings.

Reply:
The content of the messages that we receive from Pay API (in errors and warnings arrays) is out of scope for this ticket.

The presence of "following errrors" vs "following warnings" sections is out of scope for this ticket.

Please let Tracey know that if she wants to re-design this dialog and the information displayed, to please create another ticket.

Reply:
> I agree with both should use `v-html`, but the `warnings` and `errors` have different structures. `warnings` is an array of dictionaries like `[{message: '...'}, {message: '...'}]`, whereas `errors` is an array of dictionaries like `[{message: '...', payment_error_type: '...' }]`. It is the errors `message` key that contains stringified html, so v-html only applies to the errors, not the warnings.

In both cases you can iterate (for loop) over the arrays and display the "message" using v-html, right?

Reply:
Sure. I will use `v-html` for warnings too, in the same way as for the errors.

Reply:
I've changed to `v-html` for the warnings message in my last commit: 4191574.

Reply:
This conversation should not have been closed as my items 1, 2 and 4 have not been resolved to my satisfaction.

Reply:
Here are my answers to the items 1, 2, 3, 4:

1. I added to scoped style to make sure the bullets and texts are the same as the `<li>` tags. If we don't want them to look like the `<li>` tags, then we can remove the scoped style.

2. The UXA team has provided feedback on the error messages, but not on the bullets or the text alignment. If we don't use decimal pixels in css, there is a small pixel difference. If we don't want them to look like `<li>` tags, then we can remove the scoped style.

4. If we don't use custom style for the `v-icon` size, the bullets don't look the same as the `<li>` tags. 

Sorry. I am not sure what is the best option here. Please give advice when you are available. Thanks.

Reply:
We have 2 code styles for bulleted lists in this UI (and in the others):
- lists that actually use the `li` bullet
- lists that use v-icon for the bullet

In this case, for all 3 comments, unless you have specific direction from UI/UX team, make it the same as the other bulleted lists in this UI. Hint: search globally for `mdi-circle`.

Reply:
Looks good.